docs/modules/user-guide/partials/con_devfile.adoc

```diff
@@ -1,42 +1,66 @@
 [id="con_devfile_{context}"]
-= Devfile
+= About devfiles
 
 [role="_abstract"]
 
-Devfiles are YAML files that describe and define developer workspaces or cloud-native development environments. Devfiles also describe best practices for end-to-end application development and customize or extend the application development to fit a team's current development practices. Instead of forcing a new workflow or tools, devfiles provide instructions for automating manual processes.
+.How to use devfiles
 
-Development tools consume devfiles and transform the definition into a cloud workspace composed of multiple containers and cloud-native resources. Also, users can create any number of identical workspaces from the same devfile. This way, different teams can use and share workspaces in various ways, including a devfile registry or together with the source code of a project.
+Devfiles are YAML files, and they simplify your development process. In your IDE, devfiles tell you which best practices to apply to take your development project from source code to running application. Using the instructions given by devfiles, you can automate your current, manual processes. A devfile can take in the repository hosting your application source code. You can use the devfile to build your code, run your application on your local container, and then deploy your application to cloud-native containers.
 
-A devfile includes:
+.Features and benefits of devfiles
 
-* Guidance about runtime images
+Devfiles give guidelines and instructions for enhanced development processes. Devfiles use the command line interface (CLI)  tool, `odo`, to build run, and deploy your application. By using odo, devfiles provide consistent patterns of behavior for applications written in different languages and create workspaces composed of multiple containers. In these containers, you can create any number of identical workspaces from the same devfile. If you create multiple workspaces, you can share your devfile with different teams. By sharing a single devfile across multiple teams, you can ensure that each team has the same user experience and the same build, run, deploy behaviors.
+
+Using devfiles, you have the following features:
+
+* Guidance for using runtime images
 * Example code
 * Build and CI commands
 * Deployment options
 
-Benefits of devfiles:
+Use devfiles to help you with the following:
+
+* Reduce the gap between development and deployment
+* Set up a registry for finding and hosting stacks
+* Produce consistent build and run behaviors
+
+See the following example of a simple devfile:
+
+.A minimal devfile with one single project
```

Comment:
Although this is a minimal devfile example, I am not sure if this is a good example to showcase what has described above since this devfile only provides the project information without using any of the features described in the benefits list above.